app/views/admin/petitions/edit.html.erb

```diff
@@ -2,7 +2,11 @@
 
   <%= render :partial => 'petition_details', :locals => { :f => f } %>
 
-  <div>
+  <div class="row">
+    <%= link_to 'Edit petition content', admin_petition_petition_details_path(@petition) %>
+  </div>
```

Comment:
Turns out this action is kinda special and so won't appear on the "main" action list anyway.  No need to move it.